pallets/attestation/src/lib.rs

```diff
@@ -120,6 +122,12 @@ pub mod pallet {
 		type Event: From<Event<Self>> + IsType<<Self as frame_system::Config>::Event>;
 		type WeightInfo: WeightInfo;
 
+		/// The currency that is used to reserve funds for each attestation.
+		type Currency: Currency<AccountIdOf<Self>> + ReservableCurrency<AccountIdOf<Self>>;
+
+		/// The deposit that is required
+		type Deposit: Get<BalanceOf<Self>>;
```

Comment:
S: Use `#[pallet::constant]`

```suggestion
                 #[pallet::constant]
		type Deposit: Get<BalanceOf<Self>>;
```

Reply:
fixed c7f3b3c